src/main/java/com/game/Battleship/models/Board.java

```diff
@@ -0,0 +1,44 @@
+package com.game.Battleship.models;
+
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Board {
+
+    private int size;
+    private List<Cell> boardField;
+
+    public Board(){
+        this.size = 10;
```

Comment:
This is a "magic number". In cases where you have just an integer hanging out with seemingly no significance, give it some significance by attaching it to a variable. What is 10? Is it the default board size? Maybe make a class variable that communicates that.

Reply:
Done